src/components/CohortBuilder/Summary/DiagnosesChart.js

```diff
@@ -75,41 +78,81 @@ const toSingleDiagQueries = ({ topDiagnoses, sqon }) =>
     }),
   }));
 
-const DiagnosesChart = ({ topDiagnoses, sqon, theme, api, isLoading: isParentLoading }) => (
-  <QueriesResolver
-    name="GQL_DIAGNOSIS_CHART"
-    api={api}
-    queries={toSingleDiagQueries({ topDiagnoses, sqon })}
-  >
-    {({ isLoading, data }) => (
-      <CohortCard title="Most Frequent Diagnoses" loading={isLoading || isParentLoading}>
-        {!data ? (
-          <div>No data</div>
-        ) : (
-          <BarChartContainer>
-            <HorizontalBar
-              showCursor={false}
-              data={_(data)
-                .sortBy(d => d.probands + d.familyMembers)
-                .map((d, i) => ({ ...d, id: i }))
-                .value()}
-              indexBy="label"
-              keys={['probands', 'familyMembers']}
-              tooltipFormatter={mostFrequentDiagnosisTooltip}
-              tickInterval={4}
-              colors={getCohortBarColors(data, theme)}
-              xTickTextLength={28}
-              legends={[
-                { title: 'Probands', color: theme.chartColors.blue },
-                { title: 'Other Participants', color: theme.chartColors.purple },
-              ]}
-            />
-          </BarChartContainer>
+const formatedValue = value => {
```

Comment:
This formats the value to pass it to addSqon?

Reply:
yes because I receive  badly formatted data to use in my sqon

Reply:
Yes: so these are fixes needed so that the SQON searches work. Some values cannot be sent back to the backend in the same we received it from it. This doesn't affect what the user sees.

Reply:
Shouldn't we just use the `diagnoses__mondo_id_diagnosis` in the query instead?

Reply:
we use sqon not query , and where do you want to add it ?